src/applications/travel-pay/components/TravelPayFilters.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TravelPayStatusFilters from './TravelPayStatusFilters';
+
+export default function TravelPayFilters(props) {
+  const {
+    statusesToFilterBy,
+    checkedStatuses,
+    statusFilterChange,
+    applyFilters,
+    resetSearch,
+  } = props;
+
+  return (
+    <va-accordion bordered open-single>
+      <va-accordion-item
+        bordered="true"
+        data-testid="filters-accordion"
+        header="Filter travel claims"
+      >
+        <div className="filter-your-results">
+          <TravelPayStatusFilters
+            statusesToFilterBy={statusesToFilterBy}
+            checkedStatuses={checkedStatuses}
+            statusFilterChange={statusFilterChange}
```

Comment:
small nit: [it's convention](https://developer.mozilla.org/en-US/docs/Web/Events/Event_handlers#using_onevent_properties) to use `onEvent` properties for event handlers. It just makes it more explicit that it's responding to an action.
```suggestion
            onStatusFilterChange={onStatusFilterChange}
```